app/models/programme.rb

```diff
@@ -191,4 +191,12 @@ def notification_link
   def minimum_character_required_community_evidence
     0
   end
+
+  def school_achievement?
```

Comment:
Would it be better to have an `achievement_type` method, that has a raises NotImplementedError in programme, then override it in each of the specific classes to return either `:school` or `:individual`? 

I think that would give us more flexibility in the future